tests/pipeline/test_MAE.py

```diff
@@ -3,18 +3,28 @@
 
 class Test_MAE_Pipeline:
 
+    def test_pipeline_no_run(self,demo_dir):
+        LOGGER.info("run monoallelicExpression pipeline with \'run: false\'")
+        # change the third instance of "run: true" to "run: false" to turn off the MAE module
+        # run the MAE module using this config_MAE_norun (which should do nothing)
+        run("awk -v n=3 \'/run: true/ { if (++count == n) sub(/run: true/, \"run: false\"); } 1\' \
+          config.yaml > config_MAE_norun.yaml  ",demo_dir)
+        pipeline_run = run(["snakemake", "aberrantExpression", f"-j{CORES}", "--configfile", "config_MAE_norun.yaml"], demo_dir)
```

Comment:
```suggestion
        pipeline_run = run(["snakemake", "mae", f"-j{CORES}", "--configfile", "config_MAE_norun.yaml"], demo_dir)
```

Reply:
The question is why did it does not through an error? We should do a fresh start for everything.